modules/specialized-guides/pages/public-cloud-guide/payg/payg-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+== {productname} {payg} Limitations
+
+=== Bring-your-own-suscription instances already onboarded without SCC credentials
+To manage Bring-your-own-subscription (BYOS) instances in {productname} {payg} valid SCC credentials are needed. Without any SCC credentials, {productname} {payg} will not allow to onboard any BYOS, giving the user an error.
```

Comment:
> will not allow to onboard any BYOS

This is not 100% correct. This apply only for SUSE products and we have 1 exception which is "SUSE Manager Proxy BYOS". This is allowed as it requires just a management subscription which is paid via the PAYG mechanism of the Server.

Reply:
Yep, you're right. I'll change that!

Reply:
Amended!